Cargo.toml

```diff
@@ -6,3 +6,9 @@ members = [
     "tinybmp",
     "tinytga",
 ]
+
+[patch.crates-io]
```

Comment:
I asked the same question in #152 - can you explain the reasoning behind this change? Does it allow crates.io releases to work as normal, whilst enabling development against unreleased versions or something?

Reply:
It is used to use local versions for development, but I'm not sure how it's handled when uploading to crates.io.

Reply:
Ok, I'll merge #152 and release another alpha. Hopefully it doesn't have any effect.